_guide_topics/opennurbs/what-is-rhino3dmio.md

```diff
@@ -61,16 +45,35 @@ Changes that were made:
 - The project references the *Rhino3dmIO* assembly.
 - The project's *Post-build event* has been modified so the *rhino3dmio_native.dll* gets copied to the same output directory as *Rhino3dmIO.dll* when the project is built.
 
----
 
-## Mac
+
+#### Nuget package using Xamarin Studio 4.2.5 for Mac
 
 <div class="bs-callout bs-callout-danger">
   <h4>NOTE</h4>
   <p>The following process has been tested with Xamarin Studio 4.2.5 and Xcode 5.1.1. You will need to have both Xamarin Studio and Xcode installed.</p>
 </div>
 
-### Steps to compile from source
+---
+
+## Compiling Rhino3DMio from the source
```

Comment:
@dbelcher Are the "compile from source" guides entirely outdated now that we're using GYP?

Reply:
Yes, they are out of date, I just learned that.  I am removing the reference to them for right now.